tests/integration_tests.py

```diff
@@ -118,6 +118,17 @@ def build_test_list():
             "Checkpoint Integration Test - Save Load Full Checkpoint",
             "full_checkpoint",
         ),
+        OverrideDefinitions(
+            [
+                [
+                    "--checkpoint.enable_checkpoint",
+                    "--checkpoint.enable_save_safetensors_format",
+                    "--checkpoint.last_save_model_weights_only",
+                ],
+            ],
+            "Checkpoint Integration Test - Save Load Full Checkpoint",
+            "full_checkpoint_hf_safetensors",
+        ),
```

Comment:
We need to test both save in HF checkpoint and load from HF checkpoint in this test. Since we only support load weight-only initial checkpoint in HF format, I suggest the following for testing purposes only.

In the first run we save a `step-10` checkpoint and pretend it to be the initial checkpoint to load; and then in the second run we use `initial_load_path` to locate the one we saved. Something like the following, but you may have to tweak the paths a bit.
```suggestion
        OverrideDefinitions(
            [
                [
                    "--checkpoint.enable_checkpoint",
                    "--checkpoint.folder hf_checkpoint",
                    "--checkpoint.enable_save_safetensors_format",
                    "--checkpoint.last_save_model_weights_only",
                ],
                [
                    "--checkpoint.enable_checkpoint",
                    "--checkpoint.initial_load_path outputs/hf_checkpoint/step-10",
                ],
            ],
            "Checkpoint Integration Test - save load full checkpoint in HF safetensors format",
            "full_checkpoint_hf_safetensors",
        ),
```